cargo.lua

```diff
@@ -52,13 +52,15 @@ function cargo.init(config)
 
   local function halp(t, k)
     local path = (t._path .. '/' .. k):gsub('^/+', '')
-    if lf.isDirectory(path) then
+    local fileInfo = lf.getInfo(path)
+    if fileInfo and fileInfo.type == 'directory' then
```

Comment:
This could be collapsed a bit using the `FileType` constant, like `if lf.getInfo(path, 'directory') then`.  Do you want to make that change?

Reply:
Oh, sure! Coming up.